sonar-project.properties

```diff
@@ -3,7 +3,9 @@
 # SPDX-License-Identifier: MPL-2.0
 
 sonar.organization=alliander-opensource
+# TODO change to alliander-opensource_openstf
 sonar.projectKey=alliander-opensource_short-term-forecasting
```

Comment:
I think we can. I guess worst case we have to remove and add our project again and loose the sonarcloud history. But in my opinion that is not a big deal.

Reply:
Perhaps this is also useful to fix our default branch issue.

Reply:
Just deleted the old SonarCloud project and made a new one. Everything looks good so far: new name, and main as the default branch (instead of the abandoned develop).